src/fjaelllada/db_sync.py

```diff
@@ -0,0 +1,141 @@
+import json
+
+from webdav3.client import Client
+from pathlib import Path
```

Comment:
Every local file reference should be made to use `Path` instead of being a `str`.